src/scss/dp/overrides/components/_emergency-banner.scss

```diff
@@ -0,0 +1,76 @@
+.emergency-banner {
+    -webkit-font-smoothing: antialiased;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.5;
+    background-color: #b1b4b6;
```

Comment:
There are some named colours in `overrides/base/_colours.scss`. I'm not sure if you can directly reference the ones from the imported `design-system`.